tests/test_plantuml_markdown.py

```diff
@@ -0,0 +1,274 @@
+import os
+import re
+import sys
+
+import pytest
+from pytest import fixture
+
+if sys.version_info < (3, 6):
+    raise pytest.skip("plantuml_markdown plugin requires Python >= 3.6", allow_module_level=True)
+
+from tests import V8_PLUGIN_PATH, regex
+from tests.conftest import CompileResult
+from v8.plantuml_markdown.plantuml_markdown import PlantUmlMarkdownProcessor, first_line_for_listing_block
+
+
+def test_svg(do_fence_test):
+    with do_fence_test('plantuml') as compiled:
+        assert compiled.raw_html == regex(
```

Comment:
Using regex for HTML is very unclean. Better to parse it with `lxml` and query it. (For example, what if the order of `<text>` elements becomes unstable?

Reply:
You're right, xpaths make it much less brittle.